app/src/main/kotlin/com/wire/android/ui/userprofile/other/OtherUserProfileScreenViewModel.kt

```diff
@@ -309,6 +313,7 @@ class OtherUserProfileScreenViewModel @Inject constructor(
     }
 
     private suspend fun showInfoMessage(type: SnackBarMessage) {
+        _closeBottomSheet.emit(Unit)
```

Comment:
Should we always close the bottom sheet when showing the snackbar? If so maybe the name of this function should be updated to know that it doesn't just show the info message 😉

Reply:
make sense :)